detect-lang/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "codesee-detect-languages-action",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "license": "Unlicensed",
+  "dependencies": {
+    "@actions/core": "^1.10.0",
```

Comment:
Finally, I upgraded this from 1.5 to 1.10 to make the deprecation warnings go away